Frends.OpenAI.CallChatGPT/Frends.OpenAI.CallChatGPT/Frends.OpenAI.CallChatGPT.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Frends.OpenAI.CallChatGPT;
+
+using System;
+using RestSharp;
+using RestSharp.Authenticators;
+using System.ComponentModel;
+using System.Threading;
+using Definitions;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using RestSharp.Serializers.Json;
+
+/// <summary>
+/// Main class of the Task.
+/// </summary>
+public static class OpenAI
+{
+    /// <summary>
+    /// Task for calling ChatGPT API.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.OpenAI.CallChatGPT).
+    /// </summary>
+    /// <param name="input">Task input.</param>
+    /// <param name="options">Task options.</param>
+    /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+    /// <returns>object {
+    ///     bool Success,
+    ///     string Error
+    ///     object ChatCompletion {
+    ///         string Id,
+    ///         long Created,
+    ///         string Model,
+    ///         string SystemFingerprint,
+    ///         object[] Choices {
+    ///             int Index,
+    ///             object Message {
+    ///                 string Role,
+    ///                 string Content
+    ///             }
+    ///         },
+    ///     }
+    /// }</returns>
+    public static async Task<Result> CallChatGPT(
+        [PropertyTab] Input input,
+        [PropertyTab] Options options,
+        CancellationToken cancellationToken)
+    {
+        var client = CreateClient(options);
```

Comment:
RestClient is disposable class. You should use using statement with disposable classes.